libs/sdk-backend-mockingbird/src/recordedBackend/execution.ts

```diff
@@ -486,7 +486,7 @@ function normalizedDataView(
  *
  * @param recording - recording (as obtained from the index, typically using the Scenario mapping)
  * @param dataViewId - Identifier of the data view; defaults to view with all data
- * @param resultRefType - Specify what types of refs should the backend create in the result's dimension descriptors (uri refs returned by bear, id refs returned by tiger)
+ * @param resultRefType - Specify what types of refs should the backend create in the result's dimension descriptors (id refs returned by tiger)
```

Comment:
just wondering if we can remove the resultRefType fully, even from the functions as we call them. Also, it still has
```
   resultRefType: RecordedRefType = "uri",
```
as the default param value.

Reply:
Separated to STL-354